python/ray/tune/automlboard/common/exception.py

```diff
@@ -0,0 +1,8 @@
+class CollectorError(Exception):
```

Comment:
we should have a 
```
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```
on all of the files.

Reply:
ok, I've done this~